### DISCUSSION TITLE
Alt key in MacOS

### DISCUSSION THREAD

**Discussion by mettti1** [erdajt/nvim#2] [Q&A] [answered]
Love the setup, but in macos I like to use command as my Alt key.

How can i achieve the same functionality?

**Comment by erdajt** [comment, marked as answer]
Hi, thanks.

Actually I use my command key as alt too in macos for neovim.
That's why I created the [macos](https://github.com/ErdajtSopjani/nvim/tree/macos) branch, so just clone that instead of the main branch :)
